manifests/1.14/rbac/vsphere-csi-controller-rbac.yaml

```diff
@@ -15,6 +15,24 @@ rules:
   - apiGroups: ["storage.k8s.io"]
     resources: ["csidrivers"]
     verbs: ["create", "delete"]
+  - apiGroups: [""]
+    resources: ["nodes"]
```

Comment:
@andrewsykim Would love to get your eyes on the RBAC roles here...

Reply:
@codenrhoden @andrewsykim 

moved them up. 

I have also removed RBAC rules for `secrets` and `csidrivers`. so revised rules will be as below.

```
rules:
  - apiGroups: [""]
      resources: ["nodes"]
      verbs: ["get", "list", "watch"]
  - apiGroups: [""]
    resources: ["persistentvolumes"]
    verbs: ["get", "list", "watch", "update", "create", "delete"]
  - apiGroups: [""]
    resources: ["persistentvolumeclaims"]
    verbs: ["get", "list", "watch", "update"]
  - apiGroups: [""]
    resources: ["pods"]
    verbs: ["get", "list", "watch"]
  - apiGroups: ["storage.k8s.io"]
    resources: ["csinodes"]
    verbs: ["get", "list", "watch"]
  - apiGroups: ["storage.k8s.io"]
    resources: ["volumeattachments"]
    verbs: ["get", "list", "watch", "update"]
```

rules for `persistentvolumes`, `persistentvolumeclaims` and `pods` are needed for metadata syncer and full sync containers, rest all are used by controller, provisioner and attacher.

Reply:
I don't think I was clear enough on what I meant, here...

Instead of listing all these out separately, like this:

```yaml
  - apiGroups: [""]
    resources: ["secrets"]
    verbs: ["get", "list", "watch"]
  - apiGroups: [""]
    resources: ["nodes"]
    verbs: ["get", "list", "watch"]
  - apiGroups: [""]
    resources: ["persistentvolumeclaims"]
    verbs: ["get", "list", "watch"]
  - apiGroups: [""]
    resources: ["pods"]
    verbs: ["get", "list", "watch"]
```

All four of those can be combined into one:

```yaml
  - apiGroups: [""]
    resources: ["secrets", "nodes", "persistentvolumeclaims", "pods"]
    verbs: ["get", "list", "watch"]
```

Since the `apiGroup` and `verbs` are the same, you can list multiple `resources`.

Also, in the current iteration of this, it seems `secrets` have been deleted entirely. I don't think that's what you wanted.